latch/registry/table.py

```diff
@@ -265,6 +266,42 @@ def list_records(self, *, page_size: int = 100) -> Iterator[Dict[str, Record]]:
         if len(page) > 0:
             yield page
 
+    @overload
+    def get_dataframe(
+        self, *, load_if_missing: Literal[True] = True
+    ) -> Optional[pd.DataFrame]:
```

Comment:
```suggestion
    ) -> pd.DataFrame:
```

Reply:
since when load_if_missing is true its impossible to get a null return

Reply:
also idt it makes sense to have a `load_if_missing` param here, i don't see the utility of passing in `load_if_missing=False`